src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/EncodeURL.txt

```diff
@@ -0,0 +1,59 @@
+// String URL Parameter
+>> EncodeUrl("https://make.powerapps.com/")
+"https%3A%2F%2Fmake.powerapps.com%2F"
+
+// String Parameter with a character to be encoded
+>> EncodeUrl("powerfx&test")
+"powerfx%26test"
+
+// Empty String Parameter
+>> EncodeUrl("")
+""
+
+// String Parameter with a Space
+>> EncodeUrl(" ")
+"%20"
+
+// String Paremeter with multiple characters to be encoded
+>> EncodeUrl("hello!@#world")
+"hello%21%40%23world"
+
+// String Paremeter Concatenation
+>> EncodeUrl("www" & "msft.com/")
+"wwwmsft.com%2F"
+
+// String Paremeter Concatenation With Number
+>> EncodeUrl("www.msft.com/?orderid=" & 1)
+"www.msft.com%2F%3Forderid%3D1"
+
+// String Paremeter Concatenation With Boolean
+>> EncodeUrl("www.msft.com/?orderid=" & false)
+"www.msft.com%2F%3Forderid%3Dfalse"
+
+// PowerFX Parameter with Upper function
+>> EncodeUrl(Upper("www.msft.com/"))
+"WWW.MSFT.COM%2F"
+
+// PowerFX Parameter with Blank function
+>> EncodeUrl(Blank())
```

Comment:
>EncodeUrl(Blank()) [](http://example.com/codeflow?start=3&length=18)

Should this return an empty string or a Blank()? I dont have this answer but Ill find out. This is not a blocker to get this PR merged though.

Reply:
Blank inputs in string arguments should be treated as empty string, so empty string is the correct answer here.